helm_deploy/values-prod.yaml

```diff
@@ -19,6 +19,8 @@ generic-service:
     API_BASE_URL_ACTIVITIES: https://activities-api.prison.service.justice.gov.uk
```

Comment:
I think you are missing feature switches for the CASENOTE tag events for prod?

Reply:
Good point, will add next time .. im sure there will be another PR needed once i try a migration in dev